flutter/shell/platform/tizen/BUILD.gn

```diff
@@ -143,18 +143,23 @@ template("embedder") {
         "external_texture_pixel_egl.cc",
         "external_texture_surface_egl.cc",
         "flutter_platform_node_delegate_tizen.cc",
+        "tizen_autofill.cc",
         "tizen_renderer_egl.cc",
         "tizen_vsync_waiter.cc",
         "tizen_window_ecore_wl2.cc",
       ]
 
       libs += [
+        "capi-ui-autofill",
+        "capi-ui-autofill-common",
         "ecore_wl2",
         "tdm-client",
         "tizen-extension-client",
         "EGL",
         "GLESv2",
       ]
+
+      defines += [ "AUTOFILL_SUPPORT" ]
```

Comment:
https://docs.tizen.org/application/native/guides/text-input/autofill/
According to the documentation, it seems that the autoffill library can also be added to a wearable profile.
Is there a reason to exclude wearable profiles?

Reply:
The module was added in Tizen 5.5, but the wearable embedder should support Tizen 4.0 as well.

Reply:
[Check symbols #56](https://github.com/flutter-tizen/embedder/actions/runs/4131073926/jobs/7138378335)
When the first PR was opened, APIs related to autofill obtained the results of `Symbols not allowed (artifacts/libflutter_tizen_wearable.so)`. So excluded.

Reply:
@Swanseo0 By the way you can also just use the `WERABLE_PROFILE` macro without having to define another one if you want.

Reply:
@Swanseo0  Thank you for check. I also agree to use an other macro than `AUTOFILL_SUPPORT`.